canopen/objectdictionary/eds.py

```diff
@@ -36,6 +36,17 @@ def import_eds(source, node_id):
         od.node_id = int(eds.get("DeviceComissioning", "NodeID"))
 
     for section in eds.sections():
+        # Match dummy definitions
+        match = re.match(r"^[D|d]ummy[U|u]sage$", section)
+        if match is not None:
+            for i in range(1,8):
+                key = "Dummy%04d" % i
+                if eds.getint(section,key) == 1:
```

Comment:
Nitpicking a little but add a space after comma here and in the range call.

Reply:
No problem.